shoop/campaigns/migrations/0008_add_basebasketline.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def populate_new_free_product_effects(apps, schema_editor):
```

Comment:
How about just removing the last two migrations instead of doing these nasty data migrations? They're just 10 days old and I don't think anyone needs the data migration for this. Or maybe even recreate all of the campaigns migrations from scratch, since none of them existed in our last release (3.0.0). What do you think?